### PR TITLE
Add indices recovery docs

### PR DESCRIPTION
**Pull request by kovrus**

### CODE REVIEW

File: docs/sql/system.txt [hunk 1]
```diff
@@ -688,6 +688,34 @@ _node
...
+
++------------------------------------------+----------------------------------------------------+-------------+
+| Column Name                              | Description                                        | Return Type |
++==========================================+====================================================+=============+
+| ``recovery``                             | Represents recovery statistic of the particular    | ``Object``  |
+|                                          | index.                                             |             |
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['size']``                     | Statistics about physical index recovery in bytes. | ``Object``  |
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['size']['total_bytes']``      | Total number of bytes in the index.                | ``Long``    |
```
Comment (mfussenegger) [inline docs/sql/system.txt]: why are there index level information on shard level?

Comment (chaudum) [inline docs/sql/system.txt]: it should reflect https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-recovery.html

File: docs/sql/system.txt [hunk 2]
```diff
@@ -688,6 +688,34 @@ _node
 |                      | table.                           |             |
 +----------------------+----------------------------------+-------------+
 
+recovery
+--------
+
++------------------------------------------+----------------------------------------------------+-------------+
+| Column Name                              | Description                                        | Return Type |
++==========================================+====================================================+=============+
+| ``recovery``                             | Represents recovery statistic of the particular    | ``Object``  |
```
Comment (mfussenegger) [inline docs/sql/system.txt]: replace index wording with shard

File: docs/sql/system.txt [hunk 3]
```diff
@@ -688,6 +688,34 @@ _node
...
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['size']['reused_bytes']``     | Number of reused bytes for the index recovery.     | ``Long``    |
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['size']['recovered_bytes']``  | Number of actual bytes recovered in the index.     | ``Long``    |
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['files']``                    | Statistics about physical index recovery in files. | ``Object``  |
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['files']['total']``           | Total number of files in the index.                | ``Long``    |
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['files']['reused']``          | Number of reused files for the index recovery.     | ``Long``    |
```
Comment (mfussenegger) [inline docs/sql/system.txt]: describe what a reused file is

File: docs/sql/system.txt [hunk 4]
```diff
@@ -688,6 +688,34 @@ _node
...
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['files']['recovered']``       | Number of actual files recovered in the index.     | ``Long``    |
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['total_time']``               | Total time to recover the index in ms.             | ``Long``    |
```
Comment (mfussenegger) [inline docs/sql/system.txt]: Is this an estimate on how long it is expected to take for the recovery to finish or is this the amount of time the recovery has taken so far?

File: docs/sql/system.txt [hunk 5]
```diff
@@ -688,6 +688,57 @@ _node
...
+| ``recovery['type']``                     | Recovery type:                                     | ``String``  |
+|                                          |                                                    |             |
+|                                          | * store                                            |             |
+|                                          | * snapshot                                         |             |
+|                                          | * replica                                          |             |
+|                                          | * relocating                                       |             |
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['size']``                     | Shards recovery statistic in bytes.                | ``Object``  |
++------------------------------------------+----------------------------------------------------+-------------+
+| ``recovery['size']['total_bytes']``      | Total number of bytes in the shard.                | ``Long``    |
```
Comment (mfussenegger) [inline docs/sql/system.txt]: would it make sense to remove the `_bytes` suffixes? In the fs column of sys.nodes we also have just size/used

**Comment by mfussenegger** [comment]
otherwise ack

**Pull request merged**